modules/console/test/test-route.js

```
@@ -548,7 +548,7 @@ module.exports = {
             });
         });
         req.end();
-    },
+    },/* auth token expires
```

Please let us not comment tests. I think let it fail until will fix it.